src/main/java/org/spongepowered/api/world/World.java

```diff
@@ -447,4 +446,20 @@ default void setSerializationBehavior(SerializationBehavior behavior) {
      * @throws IOException If the save failed
      */
     boolean save() throws IOException;
+
+    /**
+     * Gets the view distance for this world.
+     *
+     * @return The view distance
+     */
+    int getViewDistance();
+
+    /**
+     * Sets the view distance for this world.
+     *
+     * <p>The view distance must be greater than or equal to 3, andless than or equal to 32.</p>
+     *
+     * @param viewDistance The view distance
+     */
+    void setViewDistance(final int viewDistance);
```

Comment:
Also we have no way of unsetting a world's view distance (setting it to use the global setting rather).

Reply:
as per https://github.com/SpongePowered/SpongeCommon/pull/1557#discussion-diff-142562964R107 sending a value of -1 resets it. 

**if** this remains in the API, it needs to be documented.

Reply:
we can probably do better than that